src/command_line.rs

```diff
@@ -366,6 +366,12 @@ impl Config {
             }
             Arg::Long("nosign") => self.sign = Sign::No,
             Arg::Long("nosigndb") => self.sign_db = Sign::No,
+            Arg::Long("post-build-command") => match value {
```

Comment:
TakesValue::Required handles the error if a value is not provided. You can just do `value?` to get the value.